packages/astro/src/server/meta.ts

```diff
@@ -22,9 +22,11 @@ import {
  *
  * @returns an object with the two serialized <meta> tags
  */
-export function getTracingMetaTags(span: Span | undefined, hub: Hub): { sentryTrace: string; baggage?: string } {
-  const scope = hub.getScope();
-  const client = hub.getClient();
+export function getTracingMetaTags(
```

Comment:
@Lms24 Refactored this to take a scope & client directly, this is not exported as far as I can tell?